test/userGroup.test.js

```diff
@@ -44,7 +61,12 @@ describe('/user/group', () => {
         .get('/api/user/group')
         .auth(username, password)
         .type('json')
-        .expect(HTTP_STATUS.OK);
+        .expect(function(res) {
+          if(res.status != HTTP_STATUS.OK){
```

Comment:
This can probably be removed. It looks like it was being used to debug an error.

Reply:
I left in in b/c it only logs when there is an unexpected error so there's no real downside to having it. Otherwise when this test fails you only get that the wrong status code was returned. Since when that happens it's usually an error that happens to a lot of the tests it's nice to have one print our more information. For example in this case the error I was debugging  was a mismatch in the expected public key and the one used to create the token

Reply:
For consistency sake, I would suggest removing it because no other tests do this, not even the other tests in this file. If we want to add this functionality later on we'll probably want to do it for all tests.

Reply:
I will remove it. We probably won't want to add it to all tests if we do this in the future however. It would mean adding a fair amount of code to each test and when this test fails it often means the others have failed for the same reason